blueocean-rest-impl/src/main/java/io/jenkins/blueocean/service/embedded/TryBlueOceanMenu.java

```diff
@@ -60,14 +60,14 @@
     }
 
     private boolean exists(Actionable actionable, BlueOceanUrlAction blueOceanUrlAction){
-        for(Action a: actionable.getActions()) {
-            String blueUrl  = blueOceanUrlAction.getUrlName();
-            String thisUrl = a.getUrlName();
-            if(a instanceof BlueOceanUrlAction
-                    && thisUrl != null
-                    && blueUrl != null
-                    && thisUrl.equals(blueUrl)){
-                return true;
+        for(Action a: actionable.getAllActions()) {
+            // JENKINS-44926 only call getURLName on these actions if action is BLueOceanUrlAction
```

Comment:
Ouch, yeah calling any method on a random action is dangerous. Thanks for the fix!